linode/manager
# Revert "M3-5000: Auto fill request description when opening support ticket (#7673)

## Description
This feature seems to be causing confusion for customers and Support, so this reverts commit c9da55eed3e68516b054f00bb8977c7d7858914b from https://github.com/linode/manager/pull/7673 while we figure something out.

## How to test
Try to create a Linode on an account that requires additional verification. Clicking on the `open a Support ticket` link in the Linode Create page should no longer pre-fill the description.
